test/fixtures/controller.radar/pointStyle/indexable.js

```diff
@@ -2,11 +2,11 @@ module.exports = {
   config: {
     type: 'radar',
     data: {
-      labels: [0, 1, 2, 3, 4, 5],
```

Comment:
The fixture is not changed, because `null` is skipped. You should use some other value and update the fixture(s).

Reply:
Sorry, actually the fixture is changed, and you need to add the new images to this pr.
You can use `pnpm run dev` and save the "actual" image from the failing tests (one by one) over the existing one.

Reply:
Many actual images differ from expected as of now, some are rotated some have extended widths, so should I go for it?